code/datums/status_effects/debuffs.dm

```diff
@@ -507,3 +507,29 @@
 	desc = "Your body is covered in blue ichor! You can't be revived by vitality matrices."
 	icon_state = "ichorial_stain"
 	alerttooltipstyle = "clockcult"
+
+/datum/status_effect/gonbolaPacify
+	id = "gonbolaPacify"
+	status_type = STATUS_EFFECT_MULTIPLE
+	alert_type = null
+	var/obj/item/restraints/legcuffs/bola/gonbola/gonbolaReference
+
+/datum/status_effect/gonbolaPacify/tick()
```

Comment:
I don't understand why you need to tick, remove the status effects when the gonbola is unequipped instead

Reply:
@optimumtact because the code for unequipping is even more convoluted than this.

Reply:
no its not you baby here i wrote this in like 3 minutes
https://pastebin.com/iCG3GaP5

Reply:
er fuckin dont qdel(src) at the end but delete the status effect

Reply:
nvm found a way